tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1186,6 +1016,42 @@ def test_single_sign_on_using_rhsso(enable_external_auth_rhsso, session):
         update_rhsso_settings_in_satellite(revert=True)
 
 
+@upgrade
```

Comment:
I think we don't run destructive tests for an upgrade yet. @devendra104 Thoughts ?

Reply:
@devendra104 and @jyejare I can remove them if you guys think it is not needed?

Reply:
> I think we don't run destructive tests for an upgrade yet. @devendra104 Thoughts ?

We have some destructive test cases for upgrade and that executes in every snap.